include/SDL3/SDL_platform_defines.h

```diff
@@ -475,4 +475,30 @@
 #undef __3DS__
 #endif
 
+#ifdef __NGAGE__
+
+/**
+ * A preprocessor macro that is only defined if compiling for Nintendo 3DS.
+ *
+ * \since This macro is available since SDL 3.1.3.
+ */
+#define SDL_PLATFORM_NGAGE 1
+
+typedef signed char int8_t;
+typedef unsigned char uint8_t;
+typedef signed short int16_t;
+typedef unsigned short uint16_t;
+typedef signed int int32_t;
+typedef unsigned int uint32_t;
+typedef signed long long int64_t;
+typedef unsigned long long uint64_t;
+typedef unsigned long uintptr_t;
+typedef signed long intptr_t;
+typedef long ssize_t;
+
+extern const void* nullptr;
```

Comment:
Huh?

Reply:
Can probably be removed. The Symbian API redefines `NULL` to `0`. I ultimately started using `0` whenever I had to.

Reply:
Done.